dask_expr/expr.py

```diff
@@ -208,39 +209,77 @@ def _layer(self) -> dict:
 
         return {(self._name, i): self._task(i) for i in range(self.npartitions)}
 
-    def simplify(self):
+    def simplify(
+        self,
+        phases: list
+        | tuple = (
+            "general",
+            (
+                "general",
+                "lower",
+            ),
+        ),
+    ):
```

Comment:
I would prefer using

```
(("general", ), ("general, "lower"))
```

and being more strict below

Reply:
Yeah, sounds good.